creator-node/src/diskManager.ts

```diff
@@ -112,15 +118,16 @@ function _getParentDirPathForCid(cid: string) {
  * @param {String} cid file system destination, either filename or directory
  */
 export function computeFilePath(cid: string) {
-  const parentDirPath = _getParentDirPathForCid(cid)
+  const parentDirPath = _getStorageLocationForCID(cid)
```

Comment:
nit prob change this varname to be consistent with function name change

Reply:
fixed